### DISCUSSION TITLE
Form is always reset after submit. how to make is not to

### DISCUSSION THREAD

**Discussion by egig** [ariakit/ariakit#2866] [Questions] [answered]
Hi all,

First thing, thanks to the community for such an awesome works, big one for @diegohaz, I am big fan of this.

A question, it seems like the Form is always reset to the `defaultValues` after submit, is that correct and the intended behavior ?
Is there any way to make it not to reset on submit, can't find in the `onSubmit` or `useSubmit` docs

Cheers !

**Comment by diegohaz** [comment, marked as answer]
Use the [`resetOnSubmit`](https://ariakit.org/reference/form#resetonsubmit) prop.
